### PR TITLE
Fixed typos dealing with by-index get/set in Row objects etc.

### PR DESCRIPTION
**Pull request by jbisso**
Made corrections per email. Might need to make changes in 2.2 also, because that is the branch that has API docs currently generated and posted from.

### CODE REVIEW

File: driver-core/src/main/java/com/datastax/driver/core/Row.java [hunk 1]
```diff
@@ -20,8 +20,8 @@
 /**
  * A CQL Row returned in a {@link ResultSet}.
  * <p/>
- * The values of a CQL Row can be retrieve by either index (index starts at 0)
- * or name. When setting them by name, names follow the case insensitivity
+ * The values of a CQL Row can be retrieve by either index (index starts at zero)
+ * or name. When getting or setting them by name, names follow the case insensitivity
```
Comment (adutra) [inline driver-core/src/main/java/com/datastax/driver/core/Row.java]: This really should be "When getting them" only, as it is not possible to set anything in a Row.

File: driver-core/src/main/java/com/datastax/driver/core/Row.java [hunk 2]
```diff
@@ -20,8 +20,8 @@
 /**
  * A CQL Row returned in a {@link ResultSet}.
  * <p/>
- * The values of a CQL Row can be retrieve by either index (index starts at 0)
- * or name. When setting them by name, names follow the case insensitivity
+ * The values of a CQL Row can be retrieve by either index (index starts at zero)
```
Comment (adutra) [inline driver-core/src/main/java/com/datastax/driver/core/Row.java]: Another typo: "can be retrieve**d**"

File: driver-core/src/main/java/com/datastax/driver/core/SettableByIndexData.java [hunk 3]
```diff
@@ -26,7 +26,7 @@
 import java.util.*;
 
 /**
- * Collection of (typed) CQL values that can set by index (starting a 0).
+ * Collection of (typed) CQL values that can set by index (starting at zero).
```
Comment (adutra) [inline driver-core/src/main/java/com/datastax/driver/core/SettableByIndexData.java]: "that can **be** set"

File: driver-core/src/main/java/com/datastax/driver/core/SettableData.java [hunk 4]
```diff
@@ -16,7 +16,7 @@
 package com.datastax.driver.core;
 
 /**
- * Collection of (typed) CQL values that can set either by index (starting a 0) or by name.
+ * Collection of (typed) CQL values that can set either by index (starting at zero) or by name.
```
Comment (adutra) [inline driver-core/src/main/java/com/datastax/driver/core/SettableData.java]: "that can **be** set" again – I suppose we need to amend SettableByNameData as well.

**Comment by jbisso** [comment]
Second try.

**Comment by jbisso** [comment]
One more time.

**Comment by olim7t** [comment]
:+1: LGTM

**Comment by olim7t** [comment]
Could you squash everything into one commit ?

**Pull request merged**